cmd/oras/internal/option/packer.go

```diff
@@ -120,6 +120,6 @@ func decodeJSON(filename string, v interface{}) error {
 	if err != nil {
 		return err
 	}
-	defer file.Close()
+	defer func() { _ = file.Close() }()
```

Comment:
We should return the close error

Reply:
Moved this to https://github.com/oras-project/oras/pull/1684